automation/test.sh

```diff
@@ -27,7 +27,19 @@ readonly TEMPLATES_SERVER="gs://kubevirt-vm-images"
 readonly BAZEL_CACHE="${BAZEL_CACHE:-http://bazel-cache.kubevirt-prow.svc.cluster.local:8080/kubevirt.io/kubevirt}"
 
 
-if [ ${CI} == "true" ]; then
+# Skip if it's docs changes only
+# Only if we are in CI, and this is a non-batch change
+if [[ ${CI} == "true" && -n "$PULL_BASE_SHA" && -n "$PULL_PULL_SHA" ]]; then
+    SKIP_PATTERN="^(docs/)|(OWNERS|OWNERS_ALIASES|.*\.(md|txt))$"
```

Comment:
now it skips only OWNERS (or OWNERS_ALIASES) on root isnt it ?
and we have more on non vendor, i.e 
```
./bazel/OWNERS
./cmd/OWNERS
./tests/network/OWNERS
./tests/storage/OWNERS
./tests/OWNERS
./tests/libvmi/OWNERS
./tests/performance/OWNERS
./staging/src/kubevirt.io/api/OWNERS
./OWNERS
./tools/OWNERS
./pkg/virt-operator/resource/OWNERS
./pkg/hotplug-disk/OWNERS
./pkg/network/OWNERS
./pkg/storage/OWNERS
./pkg/monitoring/OWNERS
./hack/OWNERS
```

Reply:
No, it should skip on OWNERS and OWNERS_ALIASES everywhere, which is correct.